src/main/java/io/kestra/plugin/ldap/IonToLdif.java

```diff
@@ -211,8 +224,8 @@ public IonToLdif.Output run(RunContext runContext) throws Exception {
         if (!this.inputs.isEmpty() && storedResults.isEmpty()) {
             throw new Exception("Not a single file has been translated.");
         }
-        runContext.metric(Counter.of("entries.found", this.found, "origin", "Ionise"));
-        runContext.metric(Counter.of("entries.translated", this.count, "origin", "Ionise"));
+        runContext.metric(Counter.of("entries.found", this.found, "origin", "ionise"));
+        runContext.metric(Counter.of("entries.translated", this.count, "origin", "ionise"));
```

Comment:
Could you in addition check that the origin matches the class name for every metric please?

```suggestion
        runContext.metric(Counter.of("entries.found", this.found, "origin", "IonToLdif"));
        runContext.metric(Counter.of("entries.translated", this.count, "origin", "IonToLdif"));
```

Reply:
@MilosPaunovic yeah sure noted : )

Reply:
@MilosPaunovic i just raised a pr for neo4j and have linked the issue with the pr . could you please check it - if its done right ?